packages/rocketchat-authorization/server/publications/permissions.js

```diff
@@ -20,4 +27,12 @@ Meteor.methods({
 
 RocketChat.models.Permissions.on('changed', (type, permission) => {
 	RocketChat.Notifications.notifyLoggedInThisInstance('permissions-changed', type, permission);
```

Comment:
Should this not be `notifyLogged()` it was like that earlier, so I didn't change it...